### PR TITLE
Add section about rc files

### LINKED ISSUE #250: Document our approach to configurations
This applies to tool with a config that can be specified in an `.rc` file or in the `package.json`.

Our current preference is rc files. We need to document this.

### PR DESCRIPTION
**Pull request by josebolos**
Fixes #250.

Also a bunch of unrelated whitespace fixes #sorrynotsorry

### CODE REVIEW

**Comment by josebolos** [comment]
Good point, I said "rc files" but for many tools this is going to be a json file or even a js file 🤔

**Comment by josebolos** [comment]
Rewrote the thing a bit in order to make it clear that config files can come in many ways.

File: practices/managing-node-projects.md [hunk 1]
```diff
@@ -175,6 +172,12 @@ In an ideal world where HTTP2, modern JavaScript syntax, and ES6 modules are wid
 
 As such, the fact that the asset may sometimes need to be built when serving it to a browser could be considered an accident of circumstance. It shouldn't change the classification of that dependency as a run-time dependency. In this way we maintain consistency between Node.js and web applications, and provide a clear distinction between tools that are only used in the _development_ of the application, and assets that directly contribute _functionality_ to the production application.
 
+## Configuration
+
+Some tools, like security or static analysis tools, allow us to specify a configuration for them in either configuration files in the root of the repository (usually with `.rc`, `.json`, or `.js` extensions), or as an additional entry in the `package.json` file.
```
Comment (jpw) [inline practices/managing-node-projects.md]:
```suggestion
Some tools, like security or static analysis tools, allow us to specify a configuration for them in either configuration files in the root of the repository (usually with `.{TOOL_NAME}rc`, `.json`, or `.js` extensions), or as an additional entry in the `package.json` file.
```

Comment (jpw) [reply practices/managing-node-projects.md]: maybe?

**Pull request merged**